mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -293,31 +293,31 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 							Optional:      true,
 							Computed:      true,
 							ConflictsWith: []string{"update_policy.0.max_surge_percent"},
-							Description:   `The maximum number of instances that can be created above the specified targetSize during the update process. Conflicts with max_surge_percent. If neither is set, defaults to 1`,
+							Description:   `Specifies a fixed number of VM instances. This must be a positive integer.`,
 						},
 
 						"max_surge_percent": {
 							Type:          schema.TypeInt,
 							Optional:      true,
 							ConflictsWith: []string{"update_policy.0.max_surge_fixed"},
 							ValidateFunc:  validation.IntBetween(0, 100),
-							Description:   `The maximum number of instances(calculated as percentage) that can be created above the specified targetSize during the update process. Conflicts with max_surge_fixed.`,
```

Comment:
We'll want to include `Conflicts with <-attribute->.`, this description would make sense to have if this resource had `max_surge_calculated` attribute. Which as of right now doesn't include it.

The same change will need to be done on `max_unavailable_percent` since it's missing `max_unavailable_calculated`

Reply:
Added Conflicts with to the max_surge_percent attribute